.storybook/storybook-code-source/index.js

```diff
@@ -1,28 +1,39 @@
-function tagToString(tagName, args, fromArray = false) {
-    const { body, attrs } = argsToString(args);
+function tagToString(tagName, args, plurarls = {}, indent = '', fromAttribute = false) {
```

Comment:
is `plurarls` for args to subcomponents? Maybe that name could be a little clearer

Reply:
Yes, it's basically to turn plural argument tags to singular
So `@items` becomes `@item`